lib/serviceBusMessage.ts

```diff
@@ -936,14 +936,14 @@ export class ServiceBusMessage implements ReceivedMessage {
    * rejecting the message.
    * @returns Promise<void>
    */
-  async deadLetter(options?: DeadLetterOptions): Promise<void> {
+  async deadLetter(
+    options: DeadLetterOptions = { deadletterReason: "", deadLetterErrorDescription: "" }
+  ): Promise<void> {
     let error: AmqpError = {};
-    if (options) {
-      error = {
-        condition: options.deadletterReason,
-        description: options.deadLetterErrorDescription
-      };
-    }
+    error = {
```

Comment:
Last nit :).

I think you can inline the definition of error and the setting of the properties:

```typescript
let error: AmqpError = {
  condition: options.deadletterReason,
  description: options.deadLetterErrorDescription
};
```